src/renderer/containers/app/index.jsx

```diff
@@ -7,6 +7,7 @@ import Layout from 'containers/layout';
 
 import './fonts/ProximaNova/stylesheet.scss';
 import './fonts/MaterialIcons/stylesheet.scss';
+import '../../../scss/styles.scss';
```

Comment:
minor but can we alias the path to `scss`? this would then become `import 'scss/styles.scss'`;

We would just have to add the following to webpack.config.js `scss: path.resolve(__dirname, 'renderer', 'scss'),`

Reply:
Sure, will add a commit

Reply:
This does not work as expected: 
```
@import 'scss/zindex';
^
      File to import not found or unreadable: scss/zindex.
```

This is with using this in `webpack.config.js`:
```
resolve: {
      extensions: ['.js', '.jsx', '.json', '.css', '.node', '.scss'],
      alias: {
        components: path.resolve(__dirname, 'src', 'renderer', 'components'),
        containers: path.resolve(__dirname, 'src', 'renderer', 'containers'),
        'redux-middleware': path.resolve(
          __dirname,
          'src',
          'renderer',
          'redux',
          'middleware'
        ),
        'redux-modules': path.resolve(
          __dirname,
          'src',
          'renderer',
          'redux',
          'modules'
        ),
        root: __dirname,
        scenes: path.resolve(__dirname, 'src', 'renderer', 'scenes'),
        src: path.resolve(__dirname, 'src'),
        test: path.resolve(__dirname, 'test'),
        scss: path.resolve(__dirname, 'src', 'scss'),
      },
    },
```

Reply:
None of the aliases work inside `scss` file imports actually...